riptide-parent/pom.xml

```diff
@@ -17,19 +17,26 @@
 
     <properties>
         <fauxpas.version>0.9.0</fauxpas.version>
-        <jackson.version>2.14.2</jackson.version>
-        <jackson-databind.version>2.14.2</jackson-databind.version>
-        <junit-jupiter.version>5.9.3</junit-jupiter.version>
-        <logbook.version>3.0.0</logbook.version>
-        <micrometer.version>1.10.4</micrometer.version>
-        <mockito.version>4.11.0</mockito.version>
+        <jackson.version>2.15.2</jackson.version>
+        <jackson-databind.version>2.15.2</jackson-databind.version>
+        <junit-jupiter.version>5.10.0-M1</junit-jupiter.version>
```

Comment:
any specific reason to use milestone release? is there something that you've used that isn't available in junit 5.9?